lightspeed/flame
# Add Codecov for code coverage reporting

## Description

Let's track our code coverage status with [Codecov](https://codecov.io/). Anyone will also have access to these, so :+1: for more visibility.





## How to test?

- See https://codecov.io/gh/lightspeed/flame/branch/codecov-setup for a report on this branch
- Once merged to `master`, Codecov will start comparing any new commit/branch/PR against it

## Checklist

- I have prepared [CHANGELOGs](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md#git-workflow) for release
- I have tested my changes on [supported browsers](https://browserl.ist/?q=%3E0.25%25%2C+not+op_mini+all%2C+not+ie+%3C%3D+11)
- I have added tests that prove my fix is effective or that my feature works
- When approved, I have run [visual regression tests](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md#running-visual-regression-tests), got it approved and [posted a link](https://percy.io/Lightspeed/flame)
